src/scripts/generate_cron.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This script creates the cron.yaml configuration needed to automatically update
+all the data pipelines using appengine. To generate the configuration and
+upload it to a deployed appengine application, run the following commands from
+the `src` folder, which already has an app.yaml config file:
+```sh
+python ./scripts/generate_cron.py > cron.yaml
+gcloud app deploy cron.yaml
+```
+"""
+
+import os
+import sys
+import copy
+from typing import Iterator, Dict
+import yaml
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+
+# pylint: disable=wrong-import-position
+from lib.pipeline_tools import get_pipelines
+
+
+def get_cron_jobs() -> Iterator[Dict]:
+    """ Iterator of cron job configurations """
+
+    # Define the schedules and reuse them across all applicable tasks
+    sched_hourly = {"schedule": "every 1 hours synchronized"}
+
+    # Apply sensible retry parameters for all cron jobs
+    retry_params = {"retry_parameters": {"min_backoff_seconds": 60, "max_doublings": 5}}
+
+    # Cache pull job runs hourly
+    yield {"url": f"/cache_pull", **copy.deepcopy(sched_hourly), **copy.deepcopy(retry_params)}
+
+    # The job that publishes data into the prod bucket runs every 4 hours
+    # TODO(owahltinez): disabled because it goes over the memory limit of appengine, it is being
```

Comment:
Can you include what the memory limit of appengine is here in the comment?

Reply:
Done